GA-worksheet.md

```diff
@@ -7,61 +7,78 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+ The header tag of a website holds other tags inside it like <body>.
 2. What is a Navigation Bar?
+ A navigation bar is a series of links in a website.
+3. What tag do we use to make a navigation bar? 
+ We use the <ul> tag to make a navagation bar.
 
-3. What tag do we use to make a navigation bar? <      >
-
-4.  <ul> Stands for _____________________
-    <li> Stands for ______________________
+4.  <ul> Stands for  unordered list
+    <li> Stands for  list item 
 
 5. What does the <a> tag do?
+It creates a link to another page.
 
 6. What would the line of HTML below do?
 
 ``` html
 <a href="http://www.google.com">Google</a>
 ```
+This line of html would create a link to google.com named Google.
 
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
+ 
+ Both the link tag and the style tag are ways to include CSS in an html file. The link tag in particular accesses CSS from an external file instead of keeping it all written in the html like the style tag.
```

Comment:
:+1: Why would we want to keep the HTML and CSS separate? Look up "separation of concerns."